util/src/main/scala/org/ensime/util/file.scala

```diff
@@ -6,11 +6,10 @@ import java.io.{ File => JFile, _ }
 import java.nio.charset.Charset
 import java.util.regex.Pattern
 import java.nio.file.Files
+import scala.io.Source
```

Comment:
also, you might find that a lot of these methods can just call out to their `path.scala` counterpart (in the same directory).

Ideally I'd like to remove our use of `File` and only use `Path` (actually `EnsimeFile`, which wraps `Path`)